recipes/python-seabreeze/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "python-seabreeze" %}
+{% set version = "0.6.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/ap--/python-seabreeze.git
+  git_rev: python-seabreeze-v{{ version }}
```

Comment:
We prefer not to clone repos at conda-forge. Is there a release on PyPI or a tag on GitHub that we can download a tarball from?